prod-gcp-teamfamilie.yaml

```diff
@@ -31,11 +31,11 @@ spec:
       enabled: true
   resources:
     limits:
-      memory: 1024Mi
+      memory: 500Mi
       cpu: "1"
```

Comment:
kan gjerne fjerne limits på cpu når du er igang også. ref
https://nav-it.slack.com/archives/C5KUST8N6/p1691482753195649

Reply:
Fiksa i [Fjerner cpu limit](https://github.com/navikt/familie-baks-soknad-api/pull/201/commits/6d82fcdc6a50be0ce03de92ce8e6cc419fadce10)